app/src/main/java/org/fossasia/susi/ai/MainApplication.java

```diff
@@ -23,6 +25,14 @@ public static MainApplication getInstance() {
     @Override
     public void onCreate() {
         super.onCreate();
+
```

Comment:
@amitiwary999 It is for legibility only as far as I see. Why do you want to remove these lines?